### PR TITLE
Biggest blowout ss

### PR DESCRIPTION
**Pull request by ghost**
@philjdelong @DanielEFrampton @itemniner hey guys so i refactored the biggest blowout method again. this one should be the last time we checked with Megan to make sure we are returning what we should be. Also there was that same naming issue as before that got back into our master somehow so I corrected that as well although this time I did it the hard way. if you have a moment please review and approve the new changes. thank you!!

### CODE REVIEW

File: lib/games_teams_collection.rb [hunk 1]
```diff
@@ -59,21 +59,23 @@ def percentage_ties
   end
 
   def number_of_wins
```
Comment (DanielEFrampton) [inline lib/games_teams_collection.rb:61]: Could we rename this so it reflects what it's doing now?

File: lib/games_teams_collection.rb [hunk 2]
```diff
@@ -59,21 +59,23 @@ def percentage_ties
...
+    winner_goals = []
+    @games_teams.each do |game_team|
+      if game_team.result == "WIN"
+        winner_goals << game_team.goals.to_i
       end
     end
+    winner_goals
   end
 
   def number_of_losses
```
Comment (DanielEFrampton) [inline lib/games_teams_collection.rb:71]: Could we rename this so it reflects what it's doing now?

File: test/stat_tracker_test.rb [hunk 3]
```diff
@@ -58,22 +58,22 @@ def test_it_can_count_game_by_season
...
+  def test_it_can_return_hash_of_average_goals_per_season
     expected_hash = {
                       "20122013"=>3.86,
                       "20162017"=>4.75,
                       "20142015"=>3.75,
                       "20152016"=>3.88,
                       "20132014"=>4.33
                     }
-    assert_equal expected_hash, @stat_tracker.average_goals_by_season
+    assert_equal expected_hash, @stat_tracker.average_goals_per_season
```
Comment (DanielEFrampton) [inline test/stat_tracker_test.rb]: The method name that Iteration 2 calls for is `average_goals_by_season` - if you want to discard/revert the changes on this file, my "quick fix" pull request updates it to the way the spec harness expects.

File: test/games_collection_test.rb [hunk 4]
```diff
@@ -49,7 +49,7 @@ def test_it_can_count_game_by_season
       "20132014" => 6
     }
 
-    assert_equal expected, @games_collection.count_of_games_by_season
+    assert_equal expected, @games_collection.count_of_games_per_season
   end
 
   def test_it_grabs_highest_total_score
```
Comment (DanielEFrampton) [inline test/games_collection_test.rb]: The method name that Iteration 2 calls for is `average_goals_by_season` - if you want to discard/revert the changes on this file, my "quick fix" pull request updates it to the way the spec harness expects.

File: test/games_collection_test.rb [hunk 5]
```diff
@@ -104,6 +104,6 @@ def test_it_can_return_hash_of_average_goals_by_season
                       "20152016"=>3.88,
                       "20132014"=>4.33
                     }
-    assert_equal expected_hash, @games_collection.average_goals_by_season
+    assert_equal expected_hash, @games_collection.average_goals_per_season
```
Comment (DanielEFrampton) [inline test/games_collection_test.rb:107]: The method name that Iteration 2 calls for is `average_goals_by_season` - if you want to discard/revert the changes on this file, my "quick fix" pull request updates it to the way the spec harness expects.

File: lib/stat_tracker.rb [hunk 6]
```diff
@@ -40,15 +40,15 @@ def percentage_ties
...
+  def count_of_games_per_season
+    @games.count_of_games_per_season
   end
 
   def average_goals_per_game
     @games.average_goals_per_game
   end
 
-  def average_goals_by_season
+  def average_goals_per_season
```
Comment (DanielEFrampton) [inline lib/stat_tracker.rb]: The method name that Iteration 2 calls for is `average_goals_by_season` - if you want to discard/revert the changes on this file, my "quick fix" pull request updates it to the way the spec harness expects.

**Review by DanielEFrampton — CHANGES_REQUESTED** [review]

**Review by raymondknguyen — APPROVED** [review]

File: lib/games_teams_collection.rb (hunk 1 above)
Comment (ghost) [reply lib/games_teams_collection.rb:61]: hhhhmmm it doesnt sound like what it should?

File: lib/games_teams_collection.rb (hunk 2 above)
Comment (ghost) [reply lib/games_teams_collection.rb:71]: it is finding the number of losses

**Review by DanielEFrampton — APPROVED** [review]

File: test/games_collection_test.rb (hunk 5 above)
Comment (ghost) [reply test/games_collection_test.rb:107]: done

**Review by philjdelong — APPROVED** [review]
